tests/zoomus/components/meeting/test_list_registrants.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from zoomus import components, util
+import responses
+
+
+def suite():
+    """Define all the tests of the module."""
+    suite = unittest.TestSuite()
+    suite.addTest(unittest.makeSuite(ListRegistrantsV2TestCase))
+    return suite
+
+
+class ListRegistrantsV2TestCase(unittest.TestCase):
+    def setUp(self):
+        self.component = components.meeting.MeetingComponentV2(
+            base_uri="http://foo.com",
+            config={
+                "api_key": "KEY",
+                "api_secret": "SECRET",
+                "version": util.API_VERSION_2,
+            },
+        )
+
+    @responses.activate
+    def test_can_list_registrants(self):
+        responses.add(responses.GET, "http://foo.com/meetings/ID")
+        self.component.list_registrants(id="ID")
+
+    def test_requires_id(self):
+        with self.assertRaisesRegexp(ValueError, "'id' must be set"):
+            self.component.list_registrants()
+
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
Did you run `black` to check for formatting? Looks like some files are missing a newline at the end.

Reply:
@vishnuku